ClubService.Domain/Model/Entity/Admin.cs

```diff
@@ -38,13 +38,52 @@ public class Admin
         return [domainEnvelope];
     }
     
+    public List<DomainEnvelope<IAdminDomainEvent>> ProcessMemberDeleteCommand()
+    {
+        if (Status.Equals(AdminStatus.DELETED))
```

Comment:
Here we could also use a switch case but I think we weill do this later in a refactoring so we have it in every process method.